examples/nidaqmx - time tased synchronization.py

```diff
@@ -0,0 +1,34 @@
+import nidaqmx
+import time
+from nidaqmx.constants import AcquisitionType
+from datetime import timedelta, datetime, timezone
+
+# Function to perform the measurement
+def perform_measurement():
+    with nidaqmx.Task() as task:
+        task.ai_channels.add_ai_voltage_chan("Dev1/ai0")
+        
+        # Configure finite sampling
+        num_samples = 1000
+        task.timing.cfg_samp_clk_timing(1000, sample_mode=AcquisitionType.FINITE, samps_per_chan=num_samples)

```

Comment:
With my suggested change to the read, this can be simplified

```suggestion
        num_samples = 1000
        task.timing.cfg_samp_clk_timing(1000, sample_mode=AcquisitionType.FINITE, samps_per_chan=1000)
```